ipc/provider/src/manager/evm/manager.rs

```diff
@@ -207,6 +209,12 @@ impl SubnetManager for EthSubnetManager {
             .to_u128()
             .ok_or_else(|| anyhow!("invalid min validator stake"))?;
 
+        let min_cross_msg_fee = params
```

Comment:
This should be added in dev.

Reply:
Yes, I will solve the merge conflicts in a minute.